<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyWidget</class>
 <widget class="QWidget" name="CopyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>143</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Copy file example</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Copy file:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="sourceLabel">
        <property name="text">
         <string>&amp;Source:</string>
        </property>
        <property name="buddy">
         <cstring>sourceEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="sourceEdit"/>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QPushButton" name="browseSourceButton">
        <property name="text">
         <string>&amp;Browse...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QLineEdit" name="destinationEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="destinatinLabel">
        <property name="text">
         <string>&amp;Destination:</string>
        </property>
        <property name="buddy">
         <cstring>destinationEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="browseDestinationButton">
        <property name="text">
         <string>B&amp;rowse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="runCancelButton">
     <property name="text">
      <string>&amp;Run</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
